tests/unit/kvmagent/test_virtiofs_handlers.py

```
@@ -348,47 +307,46 @@ def _make_plugin(self):
+            'mountPath': '/mnt/virtiofs/test',
         })
 
         plugin = self._make_plugin()
 
-        # Call the handler - it will fail at libvirt attach, but we just want to
-        # verify that _verify_source_path_with_recovery was called
+        # Call the handler - it will fail at later checks, but source must be ensured first.
         try:
             plugin.attach_virtiofs(req)
         except Exception:
             pass  # Expected to fail at libvirt level
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)